src/reason_lexer.mll

```diff
@@ -359,6 +386,11 @@ rule token = parse
   | "=?"
       (* Need special label extractor? *)
       { OPTIONAL_NO_DEFAULT }
+  | postfix_chars
+      {
+        let l = Lexing.lexeme lexbuf in
+        POSTFIX (remove_numbers l, int_of_string (remove_chars l))
```

Comment:
Nit: instead of using two functions, is it possible to just do one pass and return both number part and string part?